package.json

```diff
@@ -47,7 +47,14 @@
     "onCommand:rest-client.copy-response-body",
     "onCommand:rest-client.switch-environment",
     "onCommand:rest-client.clear-aad-token-cache",
-    "onLanguage:http"
+    "onLanguage:http",
```

Comment:
Minor: move the `onLanguage:http` to the item of the list

Reply:
Do you mean move `onLanguage:http` to tail of list?

Reply:
Yes 😄